src/file_pipe_log/pipe_builder.rs

```diff
@@ -223,7 +236,8 @@ impl<F: FileSystem> DualPipesBuilder<F> {
                         );
                         files.clear();
                     } else {
-                        let handle = Arc::new(self.file_system.open(&path)?);
+                        let flag = get_open_flag(self.cfg.recovery_mode, seq == max_id);
```

Comment:
I guess `seq == max_id` means this file is the active one. Is it correct? @tabokie

Reply:
yes